app/src/main/java/com/example/routinely/navigation/SetupNavGraph.kt

```diff
@@ -145,14 +184,13 @@ fun NavGraphBuilder.addTaskScreenRoute(
     onHomeButtonPressed: () -> Unit
 ) {
     composable(route = Screen.AddTaskScreen.route) {
-        AddTaskScreen(
```

Comment:
Here is the main code that should be used insted

Reply:
Done!